openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -893,6 +1344,219 @@ namespace openstudio{
     return boost::none;
   }
 
+  boost::optional<double> EpwDesignCondition::getFieldByName(const std::string &name)
```

Comment:
Should this be getFieldValueByName?  getFieldByName seems like it would be returning EpwDesignField(name);

Reply:
Again, mimicking what EpwDataPoint does.

Reply:
Ok, to me getFieldByName would return a "field" which is basically an index.  This is returning a "field value" so would be better as getFieldValueByName.  If that is the naming convention in place we can stick with it, just trying to give you my feedback.